### PR TITLE
To do list & add function with webpack

### PR DESCRIPTION
**Pull request by sntgopl**
In this milestone I did:
- Create a branch for html and css structure.
- Added modules folder with respective javascript files.
- Created arrow functions to display array of tasks in <li> elements.
- Imported all modules and functionality to index.js.
- Run npm start to start webpack live server.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,12 +1,34 @@
...
+  }
+  if (e.key === 'Enter') {
+    e.preventDefault();
+    const t = new Task(taskName.value, false, undefined);
+    t.addTask();
+    refreshList();
+    taskName.value = '';
+    return true;
+  }
+});
```
Comment (andersonlebon) [inline src/index.js]: - [ ] Kindly solve linters instead of disabling them this will perform your code.

File: src/index.js [hunk 2]
```diff
@@ -1,12 +1,34 @@
+// eslint-disable-next-line no-unused-vars
 import _ from 'lodash';
 import './style.css';
+import refreshList from './modules/functionality.js';
+import {
+  taskName,
+  // list, task, complete,
+  // description, edit, clearAll
```
Comment (andersonlebon) [inline src/index.js]: - [ ] Kindly remove commented code so that you cade look clean.

**Review by andersonlebon — CHANGES_REQUESTED** [review]
Hi @sntgopl 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.
- [ ] Kindly make sure you have a professional README file that follow the microverse README template

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omar-muhamad — COMMENTED** [review]
# Invalid ❌

Hi @sntgopl,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Your Code Review Request is marked as invalid for these reasons:

- [ ] You didn't document your work professionally:
   - You forgot to edit your README file. please use the Microverse README template.
 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

**Review by t-yanick — APPROVED** [review]
Hi @sntgopl, 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**